extensions/positron-python/python_files/positron/positron_ipykernel/inspectors.py

```diff
@@ -376,7 +380,7 @@ def value_to_json(self) -> JsonData:
         return super().value_to_json()
 
     @classmethod
-    def value_from_json(cls, type_name: str, data: JsonData) -> numbers.Number:
+    def value_from_json(cls, type_name: str, data: JsonData) -> Union[NT, numbers.Number]:
```

Comment:
`Union[NT, numbers.Number]` is sort of redundant, but pyright is unhappy with just `NT`?